pricing.js

```diff
@@ -2,35 +2,43 @@ module.exports.calculateProductPrice = function (product, employee, coverageLeve
   var price = 0
   var dollarsOff = 0
 
-  switch (product.type) {
-    case 'volLife':
-      for (var i = 0; i < coverageLevels.length; i++) {
-        var coverageAmount = coverageLevels[i].coverage
+  function volLifePrice() {
+    for (var i = 0; i < coverageLevels.length; i++) {
+      var coverageAmount = coverageLevels[i].coverage
 
-        price += (coverageAmount / product.cost.costDivisor) * product.cost.price
-      }
+      price += (coverageAmount / product.cost.costDivisor) * product.cost.price
+    }
 
-      if (product.employerContribution.mode === 'dollar') {
-        price = price - product.employerContribution.contribution
-      } else {
-        dollarsOff = price * (product.employerContribution.contribution / 100)
-        price = price - dollarsOff
-      }
+    if (product.employerContribution.mode === 'dollar') {
```

Comment:
Notice that lines 12-17 are identical to lines 26-31. We would want to DRY (don't repeat yourself) this up into it's own function that we could call in both instances rather than having it repeated.